### ISSUE TITLE
Add "first-timers-only" badge to our README

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1012]
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled.

****

http://www.firsttimersonly.com/ has a great badge for projects to advertise that they're friendly to first-timers:

![badge](http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)

It'd be great to add this to our README, right next to our "build passing" badge, linking to our new first-timers landing page (https://github.com/publiclab/plots2/projects/2)

```
![badge](http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)
```

**Comment by avi-jain** [comment]
Hello, I'd like to take this up. Will submit a PR soon
Thanks!

**Comment by jywarren** [comment]
Hello -- this is actually going to be part of the Google Code-In program,
so although we would love to have your contribution, I wonder if you might
choose one from this longer list, which is not marked `gcin` -- that would
be great!

Thank you!

**Comment by avi-jain** [comment]
Oh alright. Sure! 👍

**Comment by jywarren** [comment]
awesome 🚀

**Comment by ananyo2012** [comment]
Closed by #1042
